tests/storage/test_hotplug.py

```diff
@@ -226,7 +226,6 @@ def test_hotplug_volume_with_persist_migrate(
     ],
     indirect=True,
 )
-@pytest.mark.gating
 class TestHotPlugWithSerialPersist:
```

Comment:
please add proper quarentine marker (see how it is done for other tests, for example: `test_migration_reboot_target_node`)